scripts/polyline_processor.py

```diff
@@ -0,0 +1,46 @@
+from typing import List, Tuple
+import polyline
+import os
+from haversine import haversine
+
+IGNORE_POLYLINE = (
+    polyline.decode(os.getenv("IGNORE_POLYLINE"))
+    if os.getenv("IGNORE_POLYLINE")
+    else []
+)
+IGNORE_RANGE = int(os.getenv("IGNORE_RANGE", "0")) / 1000
```

Comment:
if IGNORE_RANGE is str it will break I wonder, do we need to info that to users?

Reply:
Yes changed.